schemas/Screen.json

```diff
@@ -28,16 +29,19 @@
         {"const": "Group", "description": "Legacy: The screen that shows information about a specific group."},
         {"const": "MyGroups", "description": "Legacy: The screen that shows all groups/communities you have joined."},
         
+        {"const": "MobileLogin", "description": "The screen that displays the login flow."},
```

Comment:
I have 2 separate thoughts on this:
- Should it be `MobileAuthentication` if it is meant to cover both login and registration flows?
- *Or* should we rename `WebLogin` → `Login` and `WebRegister` → `Register` if we want to track the 2 flows separately and use on all platforms?

Reply:
`WebLogin` → `Login` and `WebRegister` → `Register` sounds good to me.

Reply:
Updated in 5808516